values.yaml

```diff
@@ -67,7 +67,11 @@ st2:
   config: |
     [api]
     allow_origin = '*'
-
+  
+  #Override Definitions can be added here. 
+  # https://docs.stackstorm.com/latest/packs.html#overriding-pack-defaults
+  override:
+    enabled: false
```

Comment:
I'm thinking that enabled/disabled logic might be an extra option for overrides.
Unlike RBAC that's enabled/disabled in `st2.conf`, overrides are always on in st2.

Just letting the user specifying files directly might be an easier and sufficient UX for this:
```
st2:
  overrides:
    _global.yaml: |
      ---
      actions:
        defaults:
          enabled: false
    pack1.yaml: |
      # overrides for pack1
    pack2.yaml: |
      # overrides for pack2
```
https://docs.stackstorm.com/latest/packs.html#overriding-pack-defaults

@cwilson21 @cognifloyd WDYT?

Reply:
So I did the enabled/disabled option as it is a resource that is mounted that may or may not be an additional monthly/hourly charge. I am okay with either way.

Reply:
If someone wants to disable it, they should just not include any overrides files.
````
st2:
  overrides: {}
```

Reply:
Sounds good to me I will make the last changes to it when I get home today and get those into this request !

Reply:
Removed the checked for enabled, and just for if `.Values.st2.overrides.configs` exists

Reply:
That's close.
 
But instead of the following interface:
```yaml
st2:
  overrides:
    enabled: false
    config:
      _global.yaml: |
      pack1.yaml: |
```

we want to provide a simplified one, with just the override files:
```yaml
st2:
  overrides:
    _global.yaml: |
    pack1.yaml: |
```

and as @cognifloyd said, overrides could be disabled by providing an empty dict instead of `enabled` flag:
```yaml
st2:
  overrides: {}
```

Reply:
Ah okay. I misread it no problem, can get this updated should be long